__tests__/middleware-test.js

```diff
@@ -0,0 +1,158 @@
+                "numWantsSatisfied": 1,
+                "payouts": {},
+                "proposal": {}
+            },
+            "updated": "offerStatus"
+        }
+    ]
+    );
+});
+
+/**
+ * Test for check submission - with submission
+ */
+test('calls checkSubmissionForRound to check if there was a submission for round with submission', async () => {
+  
+    const submissionFound = await checkSubmissionForRound("oracle1", 1675687989744, 735);
+  
+    expect(submissionFound).toBe(true);
+});
+
+/**
+ * Test for check submission without submission
+ */
+test('calls checkSubmissionForRound to check if there was a submission for round without submission', async () => {
+  
+    const submissionFound = await checkSubmissionForRound("oracle1", 1675687989744, 736);
+  
+    expect(submissionFound).toBe(false);
+});
+
+/**
+ * Test for query price
+ */
+test('calls queryPrice to query the latest price', async () => {
+
+    readVStorage.mockReturnValue(JSON.stringify({"value": "{\"blockHeight\":\"161547\",\"values\":[\"{\\\"body\\\":\\\"{\\\\\\\"amountIn\\\\\\\":{\\\\\\\"brand\\\\\\\":{\\\\\\\"@qclass\\\\\\\":\\\\\\\"slot\\\\\\\",\\\\\\\"iface\\\\\\\":\\\\\\\"Alleged: ATOM brand\\\\\\\",\\\\\\\"index\\\\\\\":0},\\\\\\\"value\\\\\\\":{\\\\\\\"@qclass\\\\\\\":\\\\\\\"bigint\\\\\\\",\\\\\\\"digits\\\\\\\":\\\\\\\"1000000\\\\\\\"}},\\\\\\\"amountOut\\\\\\\":{\\\\\\\"brand\\\\\\\":{\\\\\\\"@qclass\\\\\\\":\\\\\\\"slot\\\\\\\",\\\\\\\"iface\\\\\\\":\\\\\\\"Alleged: USD brand\\\\\\\",\\\\\\\"index\\\\\\\":1},\\\\\\\"value\\\\\\\":{\\\\\\\"@qclass\\\\\\\":\\\\\\\"bigint\\\\\\\",\\\\\\\"digits\\\\\\\":\\\\\\\"14059528\\\\\\\"}},\\\\\\\"timer\\\\\\\":{\\\\\\\"@qclass\\\\\\\":\\\\\\\"slot\\\\\\\",\\\\\\\"iface\\\\\\\":\\\\\\\"Alleged: timerService\\\\\\\",\\\\\\\"index\\\\\\\":2},\\\\\\\"timestamp\\\\\\\":{\\\\\\\"@qclass\\\\\\\":\\\\\\\"bigint\\\\\\\",\\\\\\\"digits\\\\\\\":\\\\\\\"1676499385\\\\\\\"}}\\\",\\\"slots\\\":[\\\"board05311\\\",\\\"board02810\\\",null]}\"]}"}))
+  
+    const latestPrice = await queryPrice("ATOM-USD");
+  
+    expect(latestPrice).toBe(14.059528);
+});
+
+/**
+ * Test for get oracle invitations
+ */
+test('calls getOraclesInvitations to get the invitation IDs', async () => {
+
+    const invitations = await getOraclesInvitations("ATOM-USD");
+  
+    expect(invitations).toStrictEqual({
+        "ATOM-USD": 1675687989744
+    });
+});
+
+/**
+ * Test for query round
+ */
+test('calls queryRound to query the latest round', async () => {
+
+    readVStorage.mockReturnValue(JSON.stringify({"value": "{\"blockHeight\":\"161542\",\"values\":[\"{\\\"body\\\":\\\"{\\\\\\\"roundId\\\\\\\":{\\\\\\\"@qclass\\\\\\\":\\\\\\\"bigint\\\\\\\",\\\\\\\"digits\\\\\\\":\\\\\\\"802\\\\\\\"},\\\\\\\"startedAt\\\\\\\":{\\\\\\\"@qclass\\\\\\\":\\\\\\\"bigint\\\\\\\",\\\\\\\"digits\\\\\\\":\\\\\\\"1676499365\\\\\\\"},\\\\\\\"startedBy\\\\\\\":\\\\\\\"agoric1lw4e4aas9q84tq0q92j85rwjjjapf8dmnllnft\\\\\\\"}\\\",\\\"slots\\\":[]}\"]}"}))
```

Comment:
doesn't fit 80 char limit

Reply:
Fixed in https://github.com/jacquesvcritien/agoric-cl-middleware/commit/e7d6c899fec472591f9ed9fa8dfad2c660fbb9a5